js/main.js

```diff
@@ -30,28 +48,39 @@ function handleSubmit(event) {
   xhr.responseType = 'json';
   xhr.addEventListener('load', function () {
     var similarArtists = xhr.response.Similar.Results;
+    for (var i in similarArtists) {
+      similarArtists[i].id = data.nextArtistId;
+      data.nextArtistId++;
+    }
```

Comment:
I don't think it's necessary to apply an ID to every result, you only need to apply one when an artist is saved.